spec/requests/api/v1/user_can_search_cocktail_by_name_spec.rb

```diff
@@ -0,0 +1,52 @@
+describe 'GET /cocktails', type: :request do
+    describe'successfull' do
+        before do
+            get '/api/v1/cocktails',
+            params: {
+                q: 'Margarita'
+            }
+        end
+
+        it 'returns 200 status' do
+            expect(response.status).to eq 200
+        end
+
+        it 'returns drink' do
```

Comment:
Are you returning only one drink ? also check for count

Reply:
We can add a count test